cmd/fs-v1-metadata.go

```diff
@@ -286,16 +286,16 @@ func newFSFormatV2() (format *formatConfigV1) {
 }
 
 // Checks if input format is version 1 and 2.
-func isFSValidFormat(formatCfg *formatConfigV1) bool {
-	// Supported format versions.
-	var supportedFormatVersions = []string{
+func isKnownFSFormat(formatCfg *formatConfigV1) bool {
+	// Known format versions.
+	var knownFormatVersions = []string{
 		fsFormatV1,
 		fsFormatV2,
-		// New supported versions here.
+		// Add new versions here.
 	}
 
 	// Check for supported format versions.
-	for _, version := range supportedFormatVersions {
+	for _, version := range knownFormatVersions {
```

Comment:
If `knownFormatVersions` is `set.StringSet`, it is one liner like `return knownFormatVersions.Contains(version)`